cli/src/pixl_cli/_docker_commands.py

```diff
@@ -0,0 +1,88 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import shutil
+import subprocess
+from pathlib import Path
+from typing import Optional
+
+import click
+from decouple import config
+from loguru import logger
+
+PIXL_ROOT = Path(__file__).parents[3].resolve()
+
+docker_env_option = click.option(
+    "--env-file",
+    type=click.Path(exists=True),
+    multiple=True,
+    default=[".env"],
+    show_default=True,
+    help="Path to the .env file to use with docker compose",
+)
+docker_extra_args = click.argument("extra_args", nargs=-1, type=click.UNPROCESSED)
+
+
+# Required to allow passing unkown options to docker-compose
+# https://click.palletsprojects.com/en/8.1.x/advanced/#forwarding-unknown-options
+@click.command(context_settings={"ignore_unknown_options": True})
+@docker_env_option
+@docker_extra_args
+def up(env_file: list[Path], *, extra_args: tuple[str]) -> None:
```

Comment:
I think we might want to have this instead as `pixl dc`, then we can use all of docker compose subcommands. Then if `up` is in the args, then we add in the up args, and the check for down